system/CLI/GeneratorTrait.php

```diff
@@ -331,6 +336,8 @@ private function normalizeInputClassName(): string
     /**
      * Gets the generator view as defined in the `Config\Generators::$views`,
      * with fallback to `$template` when the defined view does not exist.
+     *
+     * @param array<string, bool|string|null> $data
```

Comment:
We can pass any type of data to views.
```suggestion
     * @param array<string, mixed> $data
```